### ISSUE TITLE
how can I test my library on an iOS6 device?

### ISSUE THREAD

**Issue by lucaquerella** [specta/specta#79]
I've written a small library (https://github.com/BendingSpoons/Stalker - developer friendly KVO and NSNotification​ interface) and I shared it on Github. 

I've written some tests using Specta, which by the way I really like. 
Now I receive an issue (https://github.com/BendingSpoons/Stalker/issues/2 - by @siutsin) asking me why I didn't support iOS 6.0. The reason is that I didn't spend time testing on iOS6 but the library should work perfectly there since I'm not using any iOS7 API.

I decide to change the deployment target, to update the .podspec, to download the iOS6 simulator and to test it there and to close the issue. When I run the test I discover that specta is built on top of XCTest which I can't use on the iOS6 simulator.  

so basically my questions are two:
1- is there any way to run my specta test on iOS6?
2- What would you do if you were me, in this situation?

**Comment by orta** [comment]
^ closing, older versions exist.
